examples/avr/src/bin/blinky.rs

```diff
@@ -0,0 +1,13 @@
+#![no_std]
+#![no_main]
+#![feature(impl_trait_in_assoc_type)]
+
+use embassy_executor::Spawner;
+use panic_halt as _;
+
+#[embassy_executor::main]
+async fn main(_spawner: Spawner) {
+    loop {
+        // TODO
```

Comment:
Toggling a pin using `avr-device` is easy enough, but the big hang up here is lack of an AVR time driver.

Is there interest in implementing/supporting an embassy-specific AVR hal, or should the examples make use of `avr-hal`? Does embassy attempt to have a consistent feel across its HALs?

One challenge is that none of the AVR HAL crates have been published, so git dependencies are needed. I'm also unsure how well it's design aligns with the HALs here.